webpack/assets/javascripts/react_app/components/PasswordStrength/PasswordStrengthSelectors.js

```diff
@@ -1,2 +1,5 @@
 export const doesPasswordsMatch = ({ password, passwordConfirmation }) =>
   !passwordConfirmation || password === passwordConfirmation;
+
+export const passwordPresent = password =>
```

Comment:
Selectors should receive the whole state tree.

I think you could do this until we will fix it:
```js
export const passwordPresent = ({ password }) => password && password.length > 0;
```